test/integration/integration.ts

```diff
@@ -93,7 +92,7 @@ async function testTransaction(
   assert.strictEqual(txData.Account, address);
   const client: Client = testcase.client;
   const signedData = client.sign(txJSON, secret);
-  console.log("PREPARED...");
+  log("PREPARED...");
```

Comment:
Can we just turn off the `console.log` rule for tests? Or is `log` doing something different from `console.log`?